### ISSUE TITLE
Add content regarding struct default initialisation

### ISSUE THREAD

**Issue by jaredkhan** [hackersatcambridge/workshop-intro-to-swift#3]

**Comment by jaredkhan** [comment]
@martinhartt Do you think we should go into what happens when you give default values to properties etc? That still confuses me a bit but it might be unnecessary?

**Comment by martinhartt** [comment]
@jaredkhan I agree that it might be confusing for the struct default initialisation. I think we should instead cover default arguments in ordinary functions, as it's quite an important feature in Swift which isn't present in many other languages (like Java). 

Once the audience will understand default args in functions, I think they can infer it to the context of struct constructors.
